src/stores/NotificationsController.ts

```diff
@@ -26,7 +26,7 @@ export const subscribeToNotifications = async (uuid: string = '') => {
           case NotificationsTypes.ACCOUNT_LOGOUT:
             await logout();
             if (notification.resource?.reason === 'sessions_limit') {
-              window.location.href = addQueryParams(window.location.href, { u: 'simultaneous-logins' });
+              window.location.href = addQueryParams(window.location.href, { u: 'login', message: 'true' });
```

Comment:
Do we not get a response from the `await login(formData.email, formData.password);` call? It would be nice not to have to rely on this. It's obscure to debug, harder to test, and risks race conditions, no?

Reply:
Can you give me example how to manage these line of code ?

Reply:
I'm not sure what the options are... it depends on the architecture of the inplayer sdk and backend, but I would assume that the login method called here: https://github.com/jwplayer/ott-web-app/blob/develop/src/containers/AccountModal/forms/Login.tsx#L21

Should be able to inspect the return value to see if the login was failed for concurrent sessions or not and set to show the appropriate notification without involving the notification controller.

Reply:
hey,

to add some context to this:

The simultaneous logins feature manages how many active sessions can have the same account( if the customer set the number greater than 0 in the dashboard ). 
On every login action, we check that number and when the limit is reached, **we invalidate the first session and send WebSocket notification.** 
The only way to know the reason for logout is to check the payload of the `account.logout` WebSocket notification, exactly like this code.

the only change that I am suggesting here is to change the `message` param( true/false ) with something like `reason` parameter;

e.g. https://..../?u=login&reason=sessions_limit
so, we can handle and validate `reason` parameter in the application.

Reply:
Ah, that makes sense. I didn't have the context and thought that this was the message on the login dialog that shows before your other sessions are logged out. I agree that an enum rather than a boolean or a raw string is good for the param type.

Reply:
d4bafdf64cf379ef4f3037c94fac29b47a6528ef